### ISSUE TITLE
Status Bar hidden at small window heights

### ISSUE THREAD

**Issue by GoogleCodeExporter** [potatomushclient/potato#50]
```
Bit of an odd behavior. The status bar gets 'scrolled' out of the active
window if the height of the window is under 600~ pixels.

To humour the System.

1. Open Potato
2. Adjust window height downward.
3. Status bar disappears.

Confirmed behavior on Windows XP, and Windows 7. The other UI elements will
do this too, but at a window height that is small enough to be irrelevant
to usability.

```

Original issue reported on code.google.com by `nullpla...@gmail.com` on 8 Dec 2009 at 7:14
- Blocked on: #98
- Merged into: #240
